### PR TITLE
graduate default pod topology spread to ga

### PR DESCRIPTION
**Pull request by kerthcet**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
graduate DefaultPodTopologySpread to GA

#### Which issue(s) this PR fixes:

REF: https://github.com/kubernetes/enhancements/issues/1258

#### Does this PR introduce a user-facing change?

```release-note
Feature of `DefaultPodTopologySpread` is graduated to GA
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
- [KEP]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/1258-default-pod-topology-spread
```

### CODE REVIEW

**Review by alculquicondor — COMMENTED** [review]
/lgtm
/approve

**Comment by kerthcet** [comment]
/retitle graduate default pod topology spread to ga

File: pkg/features/kube_features.go [hunk 1]
```diff
@@ -888,7 +888,12 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	DaemonSetUpdateSurge:                           {Default: true, PreRelease: featuregate.Beta},                    // on by default in 1.22
 	DownwardAPIHugePages:                           {Default: true, PreRelease: featuregate.Beta},                    // on by default in 1.22
 	AnyVolumeDataSource:                            {Default: false, PreRelease: featuregate.Alpha},
+<<<<<<< HEAD
```
Comment (denkensk) [inline pkg/features/kube_features.go]: It‘s better not to have this special character. Can you rebase it and fix conflict in a single commit?
You can add the `co-author` in one commit.

Comment (kerthcet) [reply pkg/features/kube_features.go]: thanks alex. done.

**Comment by denkensk** [comment]
/lgtm
Thanks. @kerthcet

**Comment by kerthcet** [comment]
Kindly ping @liggitt , PTAL.

**Pull request merged**